lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2785,15 +2785,23 @@ ParseResult FIRStmtParser::parseAssert() {
       parseExp(predicate, "expected predicate in 'assert'") ||
       parseExp(enable, "expected enable in 'assert'") ||
       parseGetSpelling(message) ||
-      parseToken(FIRToken::string, "expected message in 'assert'") ||
-      parseToken(FIRToken::r_paren, "expected ')' in 'assert'") ||
-      parseOptionalName(name) || parseOptionalInfo())
+      parseToken(FIRToken::string, "expected message in 'assert'"))
+    return failure();
+
+  SmallVector<Value, 4> operands;
+  while (!consumeIf(FIRToken::r_paren)) {
+    operands.push_back({});
+    if (parseExp(operands.back(), "expected operand in assert"))
```

Comment:
```suggestion
    if (parseExp(operands.back(), "expected operand in 'assert'"))
```

For consistency with diagnostics on other parts of parsing this.
It'd be nice to not specify the op name repeatedly like this but for now let's be consistent locally at least please.